docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+
+
+Teleport Access Monitoring allows to query and analyze the audit events generated by Teleport.
+It provides a SQL interface to query the audit logs and generate reports.
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+
+## Prerequisites
+- Teleport Cloud cluster v14.2.2 or later.
+
+## Access Monitoring RBAC Permissions
+
+To access the Access Monitoring interface, a user must have a role that allows
+`list`, `read` and `use` verbs on the `audit_query` and `audit_query` resources.
+By default the built-in `auditor` role has these permissions or you can create a custom role with these permissions e.g:
+```yaml
+kind: role
+metadata:
+  name: my-role
+spec:
+  allow:
+    rules:
+    - resources:
+      - security_report
+      verbs:
+      - list
+      - read
+      - use
+    - resources:
+      - audit_query
+      verbs:
+      - list
+      - read
+      - use
+```
+
+## Query Editor
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
+It offers a user-friendly environment for constructing SQL queries tailored to specific use cases,
+allowing users to gain deep insights into access patterns and security events within Teleport cluster.
+
+Within the Query Editor, users have access to a diverse range of SQL views representing various audit events captured by Teleport.
+These views provide a comprehensive overview of activities within the Teleport cluster, enabling users to construct precise queries tailored to their monitoring needs.
```

Comment:
This feels duplicative too.

We've already mentioned  a _powerful interface_, a _diverse range of views_, and _deep insights_. It feels like we're just repeating the same sales pitch with different words.